_rmd/2017-03-08-sperrorest-update.Rmd

```diff
@@ -0,0 +1,587 @@
+---
+layout: post
+title:  "Spatial Modeling Using Statistical Learning Techniques"
+date:  "`r format(Sys.time(), '%d %B, %Y')`"
+bibliography: "../bibs/Biblio-sperrorest.bib"
+biblio-style: apalike
+link-citations: true
+comments: true
+author: Alexander Brenning, Patrick Schratz
+categories: r
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(eval = FALSE)
+```
+
+# Introduction
+
+Geospatial data scientists often make use of a variety of statistical and machine learning techniques for spatial prediction in applications such as landslide susceptibility modeling [@Goetz2015] or habitat modeling [@Knudby2010]. Novel and often more flexible techniques promise improved predictive performances as they are better able to represent nonlinear relationships or higher-order interactions between predictors than less flexible linear models.
+
+Nevertheless, this increased flexibility comes with the risk of possible over-fitting to the training data. Since nearby spatial observations often tend to be more similar than distant ones, traditional random cross-validation is unable to detect this over-fitting whenever spatial observations are close to each other (e.g. @Brenning2005). Spatial cross-validation addresses this by resampling the data not completely randomly, but using larger spatial regions. In some cases, spatial data is grouped, e.g. in remotely-sensed land use mapping grid cells belonging to the same field share the same management procedures and cultivation history, making them more similar to each other than to pixels from other fields with the same crop type.
+
+This package provides a customizable toolkit for cross-validation (and bootstrap) estimation using a variety of spatial resampling schemes. More so, this toolkit can even be extended to spatio-temporal data or other complex data structures. This vignette will walk you through a simple case study, crop classification in central Chile [@Pena2015].
```

Comment:
which package? Is toolkit a synonym for the package? This vignette, do you mean this blog post?